### DISCUSSION TITLE
Einstellung gesucht: Auswahl des zu verwendeten Thermometers eines Thermostats in einer Gruppe

### DISCUSSION THREAD

**Discussion by sschw56** [OpenCCU/OpenCCU#2168] [🇩🇪 Fragen&Antworten] [answered]
Hi zusammen,
 
nach dem letzten Update sind ja wieder Gruppen möglich, was ich gleich genutzt habe. 
Ich habe in einem Raum zwei Thermostate. Beide senden eine IST-Temperatur, die auf Grund baulicher Tatsachen unterschiedlich sind (ein Heizkörper/Thermostat ist recht eingebaut). 
Die Gruppe verwendet nun die IST-Temperatur eines der beiden Thermostate, leider das des falschen Thermostats. 
Wie kann ich das ändern? Kann ich irgendwo bestimmen, welche IST-Temperatur von welchem Thermostat die Gruppe verwendet? 
Ich habe schon die Reihenfolge des Hinzufügens iteriert, das "falsche" Thermostat entfernt und wieder hinzugefügt etc. Immer verwendet die Gruppe das eine, "falsche" Thermostat. 
Liegt das daran, dass dieses näher am HMIP-RFUSB liegt?

Weitere Informationen:
Ich verwendet HomeAssistant auf einem Raspi 4B
und das RaspberryMatic Add On mit einem HMIP-RFUSB.

Viele Grüße!

**Comment by Baxxy13** [comment, marked as answer]
Einstellen kann man da nix.
Wer von 2 gleichberechtigten Heizkörperthermostaten der Master ist kann wohl nur eQ-3 beantworten. 
Nimm ein Wandthermostat dazu und du bist die Probleme los. Das WTH ist dann immer der Master.
